### DISCUSSION TITLE
How to filter "imageio.freeimage" warnings?

### DISCUSSION THREAD

**Discussion by ahemberger** [colour-science/colour#1054] [Q&A]
Hello;

I have a Python script I'm running in Houdini that loads an EXR image generated by Houdini (Karma); my intend is to do some simple pixel value averaging (so, in terms of Colour itself, I'm not really using it for much other than a front-end for image i/o in this case). 

I do not have openimageIO installed, and so I'm aware that Colour falls back to using imageio/freeimage to do the file opening. When it does this, I see this warning:

`imageio.freeimage warning: Warning: converting color model A/B/G/N.x/N.y/N.z/R/albedo.B/albedo.G/albedo.R to RGBA color model Warning: converting color model A/B/G/N.x/N.y/N.z/R/albedo.B/albedo.G/albedo.R to RGBA color model`

It is benign (the complaint is about extra channels in my EXR, which Houdini uses for denoising), but irritating and unhelpful for my use, so I'm trying to silence it. I notice, however, that filterWarnings doesn't currently seem to be able to filter this particular warning...I think? I wanted to confirm this, and see if it's possible from Colour to quiet it? 

Thanks!

**Comment by thomasmansencal** [comment]
Hi @ahemberger,

Did you try something along those lines: `filter_warnings(python_warnings=True)`, this should filter all the Python warnings.

Cheers,

Thomas

**Reply by ahemberger** [reply]
Thanks Thomas; I did indeed try this, unfortunately it doesn't seem to filter these particular warnings.

**Reply by thomasmansencal** [reply]
It is possible that the plugin re-enable them temporarily at this point in the code. There will not be much we can do in that case.
